src/core/utils/color/ColorValue.ts

```diff
@@ -209,7 +211,7 @@ export class ColorValue {
         hsl.h / 360,
         hsl.s / 100,
         hsl.l / 100,
-        (1 - alpha) * 255,
+        Math.round((1 - alpha) * 255),
```

Comment:
I thought we were removing all rounding? What's different about opacity?

Reply:
Is it that we want the percentage to be an integer, and then the value between 0-1 will be 2 decimal points at most?

Reply:
for RGBA all values have to be integer between 0-255